### DISCUSSION TITLE
Possible to NOT turn off light when brightness slider is at minimum?

### DISCUSSION THREAD

**Discussion by humanBeer** [piitaya/lovelace-mushroom#925] [Q&A]
Is it possible to NOT turn off the light when the brightness slider is at 0%? Many times i want the brightness to be at minimum but i end up fiddling with the slider because i keep turning it off instead, in the end i never get it to 1%. I press the icon when i want to turn it off.

**Comment by zgadson** [comment]
Seconded. This would be a very nice addition. Would be ever better if a minimum/maximum value could be set by the user.

**Comment by Fabialbo** [comment]
This is exactly what i tried to find something about the last hour. Did someone of you find a solution?

**Comment by MappacherVPN** [comment]
I would also really appreciate such a function. I was literally looking for this until I stumbled upon this thread.

**Comment by Ot4eVash** [comment]
I join! An adjustment range of 1% would be much more convenient!

**Comment by Hl2run** [comment]
> Is it possible to NOT turn off the light when the brightness slider is at 0%? Many times i want the brightness to be at minimum but i end up fiddling with the slider because i keep turning it off instead, in the end i never get it to 1%. I press the icon when i want to turn it off.

Any updates on this please? It would help me very much to set the minimum slider to be 1% and not 0% (off)

Thanks!

**Comment by mattdevo1** [comment]
Jumping in to show my support here. 

Setting a light to the "minimum" brightness without turning it off right now is very fiddly, especially on a phone/small device.
